pegasus/sites.v3/hourofcode.com/public/map.haml

```diff
@@ -6,7 +6,14 @@ layout: wide_index
 %script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&sensor=false&libraries=places,geometry&v=3.7"}
 %script{type: "text/javascript", src: "/js/jquery.geocomplete.min.js"}
 
-= view :header
+%link{href:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css', rel:'stylesheet'}
+%script{src:'https://api.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js'}
+
+  = view :header
 %h2.center-text= view :hoc_events_counter
-= view :hoc_events_map, max_height_percentage: 1
+- use_mapbox = DCDO.get("use_mapbox", false)
```

Comment:
We could also check for a URL parameter so that you can test this on production as well.

(Also, does staging use a different dataset than production?)

Reply:
Right now staging and production use the same dataset. We've got a [Jira task](https://codedotorg.atlassian.net/browse/FND-469) to create testing datasets or environments.

Added a URL param. Let me know if there's a better way to do it.

Reply:
I think that is fine for now.